lib/modules/dosomething/dosomething_settings/dosomething_settings.module

```diff
@@ -23,6 +23,25 @@ function dosomething_settings_menu() {
   return $items;
 }
 
+function dosomething_settings_init() {
+  // We don't need to redirect cli request, such as drush.
+  if (drupal_is_cli()) {
+    return;
+  }
+
+  $update_url = variable_get('dosomething_settings_realtimefeed_update_url');
+  if (!empty($update_url)) {
+    $data = ['code' => dosomething_settings_get_geo_country_code()];
+    $response = drupal_http_request($update_url, array(
```

Comment:
Will this cause a HTTP request to be made for every page load? If so, there might be performance concerns here... I think `drupal_http_request` is blocking.